views/users/landing.html

```diff
@@ -31,8 +30,8 @@ <h1 class="introduction--h1">{{title|capitalize}}</h1>
         <td data-th="Email" class="base--td">{{user.email}}</td>
         <td data-th="Role" class="base--td">{{user.role}}</td>
         <td data-th="Date" class="base--td">{{user.created}}</td>
-        <td data-th="Edit" class="base--td"><a href="{{config.base}}/{{config.actions.edit}}/{{user.id}}" class="base--link">{{config.actions.edit|capitalize}}</a></td>
-        <td data-th="Delete" class="base--td"><a href="{{config.base}}/{{config.actions.delete}}/{{user.id}}" class="base--link">{{config.actions.delete|capitalize}}</a></td>
+        <td data-th="Actions" class="base--td"><a href="{{config.base}}/{{config.actions.edit}}/{{user.id}}" class="base--link">{{config.actions.edit|capitalize}}</a></td>
+        <td data-th="" class="base--td"><a href="{{config.base}}/{{config.actions.delete}}/{{user.id}}" class="base--link">{{config.actions.delete|capitalize}}</a></td>
```

Comment:
I only need it in one so that it shows up once like this:

![image](https://cloud.githubusercontent.com/assets/6757613/17218766/9bd35b5e-54ae-11e6-9212-fcb34b309120.png)